frontend/controller/actions/identity.js

```diff
@@ -495,6 +495,7 @@ export default (sbp('sbp/selectors/register', {
     } catch (e) {
       console.error(`${e.name} during logout: ${e.message}`, e)
     }
+    sbp('gi.db/files/clear')
```

Comment:
Can you add a comment here, something like, `// clear file cache`?

Reply:
Also, maybe it makes sense to make the selector say that

Reply:
The name is to keep it consistent with the rest of the `gi.db/` selectors